docs/src/man/evaluators.md

```diff
@@ -124,23 +141,27 @@ To juggle between the mathematical description (characterized
 by a state $x$ and a control $u$) and the physical description (characterized
 by the voltage and power injection at each bus), the evaluator `nlp`
 stores internally a cache `nlp.buffer`, with type `AbstractNetworkBuffer`.
+```julia-repl
+julia> buffer = get(nlp, ExaPF.PhysicalState())
+```
 
 #### Evaluation of the callbacks
 
 Now that we have a `nlp` evaluator available, we could embed it in any
```

Comment:
Be aware that "we could" is like french "on pourrait". I think you want to say "we can". Also it may be better to use passive tense. "Now that the `nlp` evaluator is available, it can be embedded in any optimization routine".  This theme is recurring a few times.